cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -735,14 +751,18 @@ runTxBuild :: ()
   -> Maybe Word
   -> [(VotingProcedures era, Maybe (ScriptWitness WitCtxStake era))]
   -> [(Proposal era, Maybe (ScriptWitness WitCtxStake era))]
+  -> Maybe TxCurrentTreasuryValue
```

Comment:
So we are already querying the node state in `transaction build` command. What we should do is modify `queryStateForBalancedTx` to return the currency treasury value and this will allow us to avoid exposing a cli argument for the current treasury value.

`build-raw` will still need the argument however/

Reply:
@Jimbo4350> OK I see, will do :+1: